nilearn/regions/signal_extraction.py

```diff
@@ -75,45 +231,27 @@ def img_to_signals_labels(imgs, labels_img, mask_img=None,
     """
     labels_img = _utils.check_niimg_3d(labels_img)
 
-    # TODO: Make a special case for list of strings (load one image at a
-    # time).
-    imgs = _utils.check_niimg_4d(imgs)
-    target_affine = imgs.affine
-    target_shape = imgs.shape[:3]
-
-    available_reduction_strategies = {'mean', 'median', 'sum',
-                                      'minimum', 'maximum',
-                                      'standard_deviation', 'variance'}
+    available_reduction_strategies = {'mean',
```

Comment:
I think you can extract this in a `_check_reduction_strategy(strategy: str)` function

Reply:
done
also added a test that should cover this error